packages/server-core-integration/src/lib/coreConnection.ts

```diff
@@ -212,6 +212,16 @@ export class CoreConnection<
 	async createChild(coreOptions: ChildCoreOptions): Promise<CoreConnectionChild<PubSubTypes, PubSubCollections>> {
 		const child = new CoreConnectionChild<PubSubTypes, PubSubCollections>(coreOptions)
 
+		this._children.push(child)
+
+		// Set the max listeners to a higher value, so that we don't get warnings:
+		this.setMaxListeners(
```

Comment:
I'm wondering if we should do this or should we simply set it to unlimited?

I can see pros and cons both ways, so I am fine with either

Reply:
I was leaning towards that it'd be better to be conservative and keep the `maxListeners` in effect, rather than disabling it altogether.